rust/integration-tests/src/event_processor/event_processor_tests.rs

```diff
@@ -0,0 +1,137 @@
+#[cfg(test)]
+mod test {
+    use crate::{TestContext, TestProcessorConfig};
+    use aptos_indexer_test_transactions::{
+        ACCOUNT_A_GET_FUNDED_FROM_FAUCET, ACCOUNT_A_TRANSFER_TO_ACCOUNT_B,
+        ACCOUNT_B_TRANSFER_TO_ACCOUNT_B, GENERATED_USER_SCRIPT_TRANSACTION, VALIDATOR_TRANSACTION,
+    };
+    use diesel::{
+        pg::PgConnection,
+        query_dsl::methods::{FilterDsl, SelectDsl},
+        ExpressionMethods, RunQueryDsl,
+    };
+    use processor::schema::events::dsl::*;
+
+    #[tokio::test]
+    async fn test_faucet_funded_event() {
+        let test_context = TestContext::new(&[ACCOUNT_A_GET_FUNDED_FROM_FAUCET])
+            .await
+            .unwrap();
+        let processor_config = TestProcessorConfig {
+            config: processor::processors::ProcessorConfig::EventsProcessor,
+        };
+        assert!(test_context
+            .run(processor_config, |conn: &mut PgConnection| {
```

Comment:
Might be also good to pass in a reference to the test transactions so they can be used inside of the test cases.